Remotion/Web/Development.WebTesting.IntegrationTests.Infrastructure/CustomWebTestConfigurationFactory.cs

```diff
@@ -56,7 +56,7 @@ protected override IEdgeConfiguration CreateEdgeConfiguration (WebTestConfigurat
       if (string.IsNullOrEmpty (edgeVersionArchivePath))
         return new EdgeConfiguration (configSettings);
 
-      var versionedEdgeFolder = $"Edge_v{LatestTestedEdgeVersion}";
+      var versionedEdgeFolder = $"Edge_v{LatestTestedEdgeVersion}_1";

```

Comment:
Development change, need to revert before merge

Reply:
Don't forget to do this.